client/src/actions/login.coffee

```diff
@@ -0,0 +1,22 @@
+types = require '../constants/action_types.coffee'
```

Comment:
Pedantic: I like it when relative requires are at the bottom of the require statements